src/platform/elements/date-picker/DatePickerInput.ts

```diff
@@ -109,32 +107,65 @@ export class NovoDatePickerInputElement implements OnInit, ControlValueAccessor
     }
   }
 
+  _handleBlur(event: FocusEvent): void {
```

Comment:
This looks the same as handleInput except for the the extra param. you should abstract this function.

Reply:
resolved